src/main/drivers/gyro_sync.c

```diff
@@ -26,17 +26,9 @@
 
 volatile bool gyroDataReady = false;
 
-uint32_t gyroIntSignalledAt = 0;
-int32_t gyroIntSignalDelta = 0;
-
 void gyroSyncIntHandler(void)
```

Comment:
I was hoping to hear someone say that, I specifically left the file behind to get such comments 😄   It'll clean up the dependencies and reduce compile times too.